clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1722,6 +1738,11 @@ bool CompilerInvocation::ParseCodeGenArgs(CodeGenOptions &Opts, ArgList &Args,
 #include "clang/Driver/Options.inc"
 #undef CODEGEN_OPTION_WITH_MARSHALLING
 
+#define DEBUG_OPTION_WITH_MARSHALLING(...)                                     \
```

Comment:
I'd expect this to go into separate `ParseDebugArgs()` function (which is declared in the header but not defined).

Reply:
This is fixed now. I had to resolve a circular dependency between debug and codegen options for this to work.